tests/conftest.py

```diff
@@ -74,6 +75,7 @@ def run_before_each_test(
     monkeypatch: pytest.MonkeyPatch, request: pytest.FixtureRequest
 ) -> None:
     monkeypatch.setenv("PYFLUENT_TEST_NAME", request.node.name)
+    pyfluent.CONTAINER_MOUNT_PATH = pyfluent.EXAMPLES_PATH
```

Comment:
Then here we'd set the environment variable, instead of a module constant

Reply:
Done. Thank you.